demos/browser/app/meetingV2/meetingV2.html

```diff
@@ -321,6 +321,12 @@ <h1 class="h3 mb-3 font-weight-normal text-center">Select devices</h1>
           <label style="margin-left: 0.5em" for="voice-focus-setting block">Add Voice Focus</label>
         </div>
       </div>
+      <div class="row mt-3">
```

Comment:
Is there a reason this is in the preview screen and not in the first screen? All of the settings currently are in the first page.

Reply:
Ah I was going by chime app design and mainly because the first page already had a lot of settings already. I can move it to the same place as them.

Reply:
Moved the option to settings in first page